source/materials/MaterialsList.cc

```diff
@@ -98,15 +124,18 @@ namespace materials {
     return mat;
   }
 
-  G4Material* GXeDepleted(G4double pressure, G4double temperature)
-  {
-    G4double gas_density = GXeDensity(pressure);
-    G4Material* mat = GXeDepleted_bydensity(gas_density, temperature, pressure);
+  G4Material* GXeDepleted(G4double pressure, G4double temperature) {
+    std::vector<std::pair<int, double>> isotopicComposition = {
+      {124, 0.102}, {126, 0.201}, {128, 3.065}, {129, 24.900}, {130, 5.361},
```

Comment:
The composition of enriched xenon is repeated twice in the code. You define it once and reuse it.